src/material-experimental/mdc-button/button.ts

```diff
@@ -27,7 +27,15 @@ import {
   MatButtonBase
 } from './button-base';
 
-
+/**
+ * Material Design button component. Users interact with a button to perform an action.
+ * See https://material.io/components/buttons
+ *
+ * The `MatButton` class applies to native button elements and captures the appearances for
+ * "text button", "outlined button", and "contained button" per the Material Design
```

Comment:
Does it do outlined buttons? the README still mentions "Consider supporting button[mat-outlined-button]"

Reply:
We support outlined buttons, but they're _called_ `mat-stroked-button`. That TODO was about whether we want to add `mat-outlined-button` as an alias.